server/src/services/formatting/onDocumentFormatting.ts

```diff
@@ -32,9 +36,7 @@ export function onDocumentFormatting(serverState: ServerState) {
           return null;
       }
     } catch (error) {
-      serverState.logger.error(
-        `Error formatting document ${uri} with ${formatter}`
-      );
+      logger.error(`Error formatting document ${uri} with ${formatter}`);
```

Comment:
Should we not log the error?

We get logging errors in Sentry, but I think we want no errors in Sentry for failures due to syntax highlighting.

Maybe we just return error for now, and see what that looks like, and I can make modifications in later releases?

Reply:
I'll change this to info. I just realized today that the `logger.error` would trigger a sentry error.

Reply:
Agreed.